pkg/querier/querier.go

```diff
@@ -1071,6 +1071,7 @@ func containsAllIDTypes(values []string) bool {
 	return true
 }
 
+// TODO(twhitney): Delete this method and the GRPC service signature. This is now handled in the query frontend.
```

Comment:
why not delete in this PR?

Reply:
I don't want to change the GRPC interface in this PR, as that could complicate roll-outs. Without changing the interface, we need to implement something, so I figure might as well keep it until the PR when we modify the interface.